### PR TITLE
AA: Add Optimism/Arbitrum mainnet and testnet

### PR DESCRIPTION
**Pull request by alberto-crossmint**
## Description

Adding two different chains with low gas fees and EIP-7212 support

Some highlights:
- Set the webhook policy to the same as for other SFK projects. That is, 
`https://crossmint-main-staging.azurewebsites.net/api/unstable/wallets/aa/webhookPolicy`
`https://crossmint-main.azurewebsites.net/api/unstable/wallets/aa/webhookPolicy`
- Set the gas sponsoring policy to 500 gwei max on mainnets and 1000 gwei on testnets

Decided not to integrate Ethereum as neither gas is cheap nor has secp256r1 support.

Both chains support the latest v0.7 Entrypoint and v3.1 Kernel

### Minting test

AA mint on Optimism using passkeys https://sepolia-optimism.etherscan.io/tx/0x2e4cf72c81b0936f5dd06cf5a35f0637ade9e49dd9b87acd299b8d83b766b7f0 

And another AA mint on Arbitrum Sepolia https://sepolia.arbiscan.io/tx/0xe1ba6cf7bafff96d9a9977db1a90d549cb3f860ff2352971467de6008fc57646 

### Signature test

Using this code

```typescript
    const signedData = await account.client.wallet.signMessage({
        message: `Mysignedmessage`,
    })
    const verificationResult = await account.client.public.verifyMessage(
        {
            address: account.address,
            message: `Mysignedmessage`,
            signature: signedData,
        }
    )
    console.log("Verification result", verificationResult);
```

It properly verifies the message in
- Arbitrum Sepolia ✅ 
- Optimism Sepolia ✅ 

## Follow up

Adapt demo to also use these chains

### CODE REVIEW

File: packages/client/wallets/smart-wallet/src/blockchain/chains.ts [hunk 1]
```diff
@@ -29,13 +43,21 @@ export const zerodevProjects: Record<SmartWalletChain, string> = {
     "polygon-amoy": "3deef404-ca06-4a5d-9a58-907c99e7ef00",
     "base-sepolia": "5a127978-6473-4784-9dfb-f74395b220a6",
     base: "e8b3020f-4dde-4176-8a7d-be8102527a5c",
+    "optimism-sepolia": "f8dd488e-eaed-467d-a5de-0184c160f3b1",
```
Comment (MatiNavCrossmint) [inline packages/client/wallets/smart-wallet/src/blockchain/chains.ts:46]: I'm curious that we weren't supporting this. I remember testing these chains some months ago, but probably something changed.

**Review by MatiNavCrossmint — APPROVED** [review]
LGTM! Just curious that we didn't support these chains before

File: packages/client/wallets/smart-wallet/src/blockchain/chains.ts (hunk 1 above)
Comment (alberto-crossmint) [reply packages/client/wallets/smart-wallet/src/blockchain/chains.ts:46]: The main repo supports them out of the box as the default contract version configuration works for it and we have ZeroDev projects configured for it, it was just the SDK was never adapted so these are supported chains. Here we create SDK-specific ZeroDev projects and add these chains to the SDK

Comment (afeight) [reply packages/client/wallets/smart-wallet/src/blockchain/chains.ts:46]: @MatiNavCrossmint I think we dropped support for them at one point to just say polygon & base work

**Review by afeight — APPROVED** [review]
looks good, just check with Nischay about the 500 limit

**Review by bryan-eastwood3 — APPROVED** [review]

**Pull request merged**